x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java

```diff
@@ -251,5 +255,14 @@ public void testWithOnlyTrainingRowsAndTrainingPercentIsFifty() throws Exception
             "Creating destination index [regression_only_training_data_and_training_percent_is_fifty_source_index_results]",
             "Finished reindexing to destination index [regression_only_training_data_and_training_percent_is_fifty_source_index_results]",
             "Finished analysis");
+        assertModelStatePersisted(jobId);
+    }
+
+    private void assertModelStatePersisted(String jobId) {
```

Comment:
Could this be `static`?

Reply:
I will make it static in a different PR to avoid wasting CI cycles